daemon/src/peer/gossip.rs

```diff
@@ -26,8 +26,8 @@ where
         rev: rev.map(|rev| Rev::Git(rev.into())),
         origin: None,
     }) {
-        Ok(()) => log::trace!("successfully announced for urn=`{}`, rev=`{:?}`", urn, rev),
-        Err(_payload) => log::warn!("failed to announce for urn=`{}`, rev=`{:?}`", urn, rev),
+        Ok(()) => tracing::trace!(?urn, ?rev, "successfully announced URN"),
```

Comment:
```suggestion
        Ok(()) => tracing::trace!(%urn, ?rev, "successfully announced URN"),
```

`Urn`s are easier to debug with their `Display`s being used :)

Reply:
Fixed in e0f8ba665088506a52784ac5335cbb639d40818b